SPECS/kernel/config_aarch64

```diff
@@ -540,18 +579,25 @@ CONFIG_ARM_PSCI_FW=y
 #
 # EFI (Extensible Firmware Interface) Support
 #
-# CONFIG_EFI_VARS is not set
+CONFIG_EFI_VARS=y
 CONFIG_EFI_ESRT=y
+CONFIG_EFI_VARS_PSTORE=m
+# CONFIG_EFI_VARS_PSTORE_DEFAULT_DISABLE is not set
+CONFIG_EFI_SOFT_RESERVE=y
 CONFIG_EFI_PARAMS_FROM_FDT=y
 CONFIG_EFI_RUNTIME_WRAPPERS=y
 CONFIG_EFI_ARMSTUB=y
 CONFIG_EFI_ARMSTUB_DTB_LOADER=y
-# CONFIG_EFI_CAPSULE_LOADER is not set
-# CONFIG_EFI_TEST is not set
+CONFIG_EFI_BOOTLOADER_CONTROL=m
+CONFIG_EFI_CAPSULE_LOADER=m
+CONFIG_EFI_TEST=m
 CONFIG_RESET_ATTACK_MITIGATION=y
 # end of EFI (Extensible Firmware Interface) Support
 
+CONFIG_UEFI_CPER=y
+CONFIG_UEFI_CPER_ARM=y
 CONFIG_EFI_EARLYCON=y
+CONFIG_EFI_CUSTOM_SSDT_OVERLAYS=y
```

Comment:
This config has been identified by the kernel config hardening checker and should be disabled

Reply:
Discussed offline. May be required for boot. Will prune down in future PR.